src/main/java/lotto/domain/LottoGames.java

```diff
@@ -0,0 +1,53 @@
+package lotto.domain;
+
+import lotto.domain.generator.AutoLottoGenerator;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+public class LottoGames {
+
+    private final List<Lotto> lottoGameList = new ArrayList<>();
+    private final List<Integer> lottoResult = Arrays.stream(new int[LottoConstant.LOTTO_SIZE + 1]).boxed().collect(Collectors.toList());
+
+    public LottoGames(int gameCount) {
+        gameCount /= LottoConstant.LOTTO_PRICE;
+        for (int i = 0; i < gameCount; i++) {
+            lottoGameList.add(new Lotto(new AutoLottoGenerator()));
+        }
```

Comment:
```suggestion
        this(lottoGameList);
```

이 생성자는 부 생성자이다.
부 생성자는 주 생성자를 통해 초기화하는 습관을 들이면 어떨까?